calm-ingest/prototype.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
```

Comment:
Do we need to keep track of where we are in the ingest from Dynamo to ES? This is ok to merge for now, but can we chat about how this step is going to work when we start getting updates rather than batch processing old stuff.